www/src/public/js/controllers/builder/main.js

```diff
@@ -1517,7 +1541,10 @@
                     if (totalBaseStats < 244) {
                         fromStatQuests += 3;
                     }
-                    if ($scope.selectedList.baseStats.amulet == 1) {
+                    if ($scope.selectedList.baseStats.amulet == 1 || $scope.selectedList.baseStats.hazelnut == 1) {
```

Comment:
if hazelnut == 1 and amulet != 1, it'll still increase this stat by 20, because of the or in the first if

Reply:
Good catch, I had caught this error and corrected it originally but either this one snuck past me or I forgot to save.

Fixed!